app/_hub/kong-inc/rate-limiting/index.md

```diff
@@ -157,7 +157,7 @@ There are 2 use cases that are most common:
 **NOTE**:
 
 <div class="alert alert-warning">
-  <strong>Enterprise-Only</strong> The Kong Community Edition of this Rate Limiting plugin does not
+  <strong>Enterprise-Only</strong> The Kong of this Rate Limiting plugin does not
```

Comment:
Wording is now broken (it worked with "Edition" present, but is incorrect grammar without it)